docs/supported.md

```diff
@@ -91,3 +93,60 @@ Earlier and Later versions might be only partially supported, we don't fully kno
+
+ISR pages are built by the Vercel CLI to something called prerender functions. These are Node.js Lambda functions that can be called in the background while serving the page from the cache. It is not possible to use these with Cloudflare Pages and they are not compatible with the edge runtime.
+
+next-on-pages will use the static fallback files for your ISR/prerendered pages.
+
+### Statically Generated Routes Edge Cases
+
+Next.js performs a form of static analysis on routes to determine whether they can be [statically generated](https://nextjs.org/docs/app/building-your-application/rendering/static-and-dynamic-rendering), or if they need to be dynamically rendered at request time. In most cases, next-on-pages can handle these pages fine. However, there are some edge cases where this is not possible with pages that have dynamic parameters.
+
+#### `generateStaticParams`
+
+When doing static site generation (SSG) in the app directory, you need to tell the page that it is not possible for non-prerendered dynamic routes to occur, i.e. [they should 404](https://nextjs.org/docs/app/api-reference/file-conventions/route-segment-config#dynamicparams). This is so that it doesn't generate a Node.js Lamba function for the dynamic parameter, while generating prerendered HTMl files for all params provided by `generateStaticParams`.
+
+```diff
++ export const dynamicParams = false;
+```
+
+#### `getStaticPaths`
+
+In the pages directory, when doing static site generation (SSG), there is a similar scenario as there was with `generateStaticParams`. Because of how ISR functions work, you need to tell the page that [it should 404](https://nextjs.org/docs/pages/api-reference/functions/get-static-paths#fallback-false) for non-pregenerated dynamic routes. This is done by setting `fallback: false` in `getStaticPaths`.
+
+```diff
+export async function getStaticPaths() {
+	const res = await fetch('https://.../posts');
+	const posts = await res.json();
+
+	// Get the paths we want to pre-render based on posts
+	const paths = posts.map(post => ({ params: { id: post.id } }));
+
+	// We'll pre-render only these paths at build time.
+	// { fallback: false } means other routes should 404.
+	return {
+		paths,
++		fallback: false.
+	};
+}
+```
+
+### Revalidating Data and `next/cache`
+
+Revalidation and `next/cache` are not supported on Cloudflare Pages due to how they are implemented in Next.js. They appear to depend on Vercel infrastructure to deal with something referred to as the [suspense cache](https://github.com/vercel/next.js/blob/canary/packages/next/src/server/lib/incremental-cache/fetch-cache.ts#L38). This is used by the default fetch cache, which forms part of the incremental cache for revalidation.
+
+Revalidating tags and paths goes through this functionality and appears to be what `next/cache` is built around. Because of this, it is highly unlikely that the default `next/cache` functionality will be supported on Cloudflare Pages in the future.
```

Comment:
```suggestion
Revalidating tags and paths goes through this functionality and appears to be what `next/cache` is built around. Because of this, it is highly unlikely that the default `next/cache` functionality will be supported on Cloudflare Pages.
```